src/src/pages/capabilities/SelectedCapabilityContext.js

```diff
@@ -349,22 +357,30 @@ function SelectedCapabilityProvider({ children }) {
   }, [isLoadedMembers, membersList]);
 
   useEffect(() => {
-    if (details) {
-      loadMembershipApplications();
-      loadKafkaClustersAndTopics();
-      loadAwsAccount();
-    } else {
-      setMembers([]);
-      setMembershipApplications([]);
-      setKafkaClusters([]);
+    if (isLoadedMembersApplications) {
+      setMembershipApplications(membersApplicationsList);
     }
-  }, [details]);
+  }, [isLoadedMembersApplications, membersApplicationsList]);
+
+
+
+  // useEffect(() => {
+  //   if (details) {
+  //     //  loadMembershipApplications();
+  //     //loadKafkaClustersAndTopics();
+  //     // loadAwsAccount();
+  //   } else {
+  //     setMembers([]);
+  //     setMembershipApplications([]);
+  //     setKafkaClusters([]);
+  //   }
+  // }, [details]);
 
   // setup reload of kafka clusters and topics
   useEffect(() => {
     const handle = setInterval(() => {
       if (details && shouldAutoReloadTopics) {
-        loadKafkaClustersAndTopics();
+        //loadKafkaClustersAndTopics();
       }
     }, 5 * 1000);
```

Comment:
This means that we no longer have the functionality where we keep polling for new data. Is it something we want to remove or?